cli/targets/static.js

```diff
@@ -690,13 +730,32 @@ function buildService(ref, service) {
             "@function " + lcName,
             "@memberof " + exportName(service),
             "@instance",
-            "@param {" + exportName(method.resolvedRequestType, !config.forceMessage) + "} request " + method.resolvedRequestType.name + " message or plain object",
-            "@returns {Promise<" + exportName(method.resolvedResponseType) + ">} Promise",
+            "@param {" +  exportTypeWithProtoFromIfNeeded(method.resolvedRequestType, !config.forceMessage) + "} request " + method.resolvedRequestType.name + " message or plain object",
+            "@returns {Promise<" + exportTypeWithProtoFromIfNeeded(method.resolvedResponseType) + ">} Promise",
             "@variation 2"
         ]);
     });
 }
 
+function exportTypeWithProtoFromIfNeeded(type, asInterface) {
```

Comment:
I recommend keeping function names more succinct. `ifNeeded` looks excess and can be casted aside
```suggestion
function exportTypeWithProtoFrom(type, asInterface) {
```